rewrite-groovy/src/main/java/org/openrewrite/groovy/GroovyParserVisitor.java

```diff
@@ -1374,7 +1374,7 @@ public void visitConstantExpression(ConstantExpression expression) {
         public void visitConstructorCallExpression(ConstructorCallExpression ctor) {
             queue.add(insideParentheses(ctor, fmt -> {
                 cursor += 3; // skip "new"
-                TypeTree clazz = visitTypeTree(ctor.getType(), ctor.getMetaDataMap().containsKey(StaticTypesMarker.INFERRED_TYPE));
+                TypeTree clazz = visitTypeTree(ctor.getType(), ctor.getNodeMetaData().containsKey(StaticTypesMarker.INFERRED_TYPE));
```

Comment:
https://github.com/openrewrite/rewrite/pull/4742/files#diff-d900151f599dd809ec1076aca6e0ecb67997b3b5b1e52e435d1c64de318fd834R1383
Uses the same way of retrieving the metadata